### PR TITLE
fix(Fieldset): css alignments

### PR DESCRIPTION
**Pull request by eirikbacker**
Part of #2295

### CODE REVIEW

File: packages/css/fieldset.css [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+    display: inline-flex;
+  }
+
+  & > legend:empty {
+    display: none;
+  }
+
+  &[data-hidelegend] > legend {
+    @composes ds-sr-only from '../css/utilities.css';
+  }
```
Comment (Barsnes) [inline packages/css/fieldset.css:24]: This does not hide the description, as it does currently.
Should this change maybe be done in another PR?

Comment (eirikbacker) [reply packages/css/fieldset.css:24]: oh right! Sorry was experimenting and forgot to bring back. I'll fix, well spotted ☺️ 
I think we should just have in the same PR 👍

Comment (Barsnes) [reply packages/css/fieldset.css:24]: If we agree to keep description when legend is hidden, I'm all for keeping it in this PR - but that might take some time to decide on

**Review by Barsnes — APPROVED** [review]
🚀
